src/ios/view.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * test view
+ *
+ * Created by Appcelerator, Inc
+ * Copyright (c) 2016 Appcelerator. All rights reserved.
+ */
+import { UIScreen } from 'UIKit/UIScreen';
+import { UIColor } from 'UIKit/UIColor';
+import { UIView } from 'UIKit/UIView';
+
+export default class View {
+	constructor (opts = {}) {
```

Comment:
I'm not a fan of putting a space between the function name and the `(`, especially in ES6 classes. I've never seen anybody that does that.